iothub/service/src/Utilities/ClientApiVersionHelper.cs

```diff
@@ -14,7 +14,7 @@ internal class ClientApiVersionHelper
 
         /// <summary>
         /// The default API version query string parameter to use for all data-plane service calls in <see cref="RegistryManager"/>,
-        /// <see cref="ServiceClient"/>, and <see cref="JobClient"/>.
+        /// <see cref="ServiceClient"/>, and <see cref="ScheduledJobsClient"/>.
```

Comment:
I'd go ahead and delete all the references to specific clients here since we will now have many more.

"The API version used in all service requests." is sufficient here